src/BTVNanoCommissioning/utils/histogrammer.py

```diff
@@ -473,18 +540,28 @@ def histogrammer(events, workflow):
     disc_list = [
         "btagDeepFlavB",
         "btagDeepFlavC",
+        "btagTransDeepFlavB",
+        "btagTransDeepFlavC",
         "btagDeepFlavCvL",
         "btagDeepFlavCvB",
         "btagDeepFlavB_b",
         "btagDeepFlavB_bb",
+        "btagTransDeepFlavB_lepb",
+        "btagTransDeepFlavB_b",
+        "btagTransDeepFlavB_bb",
+        "btagTransDeepFlavB_lepb",
         "btagPNetB",
+        "btagTransPNetB",
         "btagPNetCvB",
         "btagPNetCvL",
-        "btagPNetCvNotB",
```

Comment:
This should not removed, please add this back

Reply:
Added back